alert.css

```diff
@@ -0,0 +1,37 @@
+@import url(components.css);
+.content{
+    margin: 10px;
+    padding: 10px;
+    font-size: 1.5rem;
+    font-weight: 700;
+    text-align: center;
+}
+.content-background
+{
+    height: fit-content;
+    width: 900px;
+    border-radius: 15px;
+    box-shadow:5px 10px 8px #888888;
+    padding: 20px;
+    margin-bottom: 10px;
+}
+   .content.content-background.D
+{
+    background-color: var(--danger);
+}
+.content.content-background.S
+{
+   background-color: var(--success);
+  
```

Comment:
remove extra spacing and get consistency in the bracket structure

Reply:
done removing extra spaces